f/fluentd/build_info.json

```diff
@@ -0,0 +1,22 @@
+{
+    "package_name" : "fluentd",
+    "github_url":"https://github.com/fluent/fluentd",
+    "package_dir": "f/fluentd/Dockerfiles/",
+    "build_script": "fluentd_ubi_8.5.sh",
+    "docker_cmd": "docker build -t ${package_name}:$PACKAGE_VERSION ${dir}",
+
+    "v0.12.43" : {
+        "dir":"1.14.5_es7.17.2_ubi8"
+    },
+
+    "v1.14.2" : {
+        "dir":"1.14.5_es7.17.2_ubi8"
+    },
+    "v1.14.5" : {
+        "dir":"1.14.5_es7.17.2_ubi8"
+    },
+
+    "v1.14.*" : {
+        "dir":"1.14.5_es7.17.2_ubi8"
```

Comment:
All are pointing to the same directory and there are no extra build arguments.
I think, we can remove below:
```
    "v1.14.2" : {
        "dir":"1.14.5_es7.17.2_ubi8"
    },
    "v1.14.5" : {
        "dir":"1.14.5_es7.17.2_ubi8"
    },
```